pontoon/base/static/js/fluent_interface.js

```diff
@@ -57,43 +57,55 @@ var Pontoon = (function (my) {
         }
         // Plurals
         else if (self.isPlural(entityAST)) {
+          function renderPluralEditor(value, name, pluralIndex) {
```

Comment:
Functions inside functions are generally a bad pattern. Can you move that to a different location? Ideally, outside of the `Pontoon` object, it already has too much going on. :)

Reply:
`Pontoon.fluent` is actually the only meaningful option I can think of. Do you have other ideas?

Reply:
You don't need that function outside of the file, so it can simply be at the top of the file, in the closure.